docs/source/gbd2017_models/concept_models/vivarium_sanofi_zenon/index.rst

```diff
@@ -0,0 +1,270 @@
+
+2019 Guidelines with multiple pills scenario
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For each of the simulated populations/locations, IHME will run a counterfactual scenario in which all initial variables are identical to the BAU Scenario, but in which treatments follow the ESC 2019 Guidelines using individual components (i.e., multiple pills) for individuals meeting guideline criteria. The same treatment pathway will be applied to all countries based on ESC 2019 Guidelines. The guidelines propose a more proactive treatment than current treatment practices. More information about this scenario and associated adherence to multiple pills treatment can be found in the Treamtent Ramp below and is included in the Data Artifact.
+
+2019 Guidelines with combination single pill 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For each of the simulated populations/locations, IHME will run a counterfactual scenario in which all initial variables are identical to '2019 Guidelines with multiple pills scenario', but in which treatments follow the ESC 2019 Guidelines using FDC rather than individual components for individuals meeting guideline criteria. This scenario would include an assumption about the increased adherence to single pill treatment compared to multiple pills. More information about this scenario and the increased adherence to single pill treatment can be found in the Treamtent Ramp below and is included in the Data Artifact.
+
+Questions of Interest
++++++++++++++++++++++
+
+
+Scope of Modeling
++++++++++++++++++
+
+
+Concept Model Diagram
+---------------------
+
+.. image:: zenon_concept_model_diagram.svg
+
+Model Components
+----------------
+
+Time
+++++
+
+* Start and end date: **Jan 1, 2020 -- Dec 31, 2024**
+* Simulation time step: **28 days** to capture cardiovascular events and treatment timesteps
+
+Demographics
+++++++++++++
+
+* Locations: **Brazil, China, France, Italy, Spain, Russia**
+* Population: **Prospective open cohort of 40-74 year olds**
+* Size of largest starting population: **100,000 simulants**
+* Youngest start-age and oldest end-age: **30 - 100 years**
+* Exit age (at what age to stop tracking simulants): **101 years**
+* Fertility: **Not applicable**
+
+GBD Causes
+++++++++++
+
+* :ref:`Ischemic Heart Disease <2017_cause_ischemic_heart_disease>`
+
+* :ref:`Ischemic Stroke <2017_cause_ischemic_stroke>`
+
+GBD Risks
++++++++++
+
+* :ref:`High LDL cholesterol <2017_risk_high_ldl_c>`
+
+PAF-of-1 Cause/Risk Pairs
++++++++++++++++++++++++++
+
+* :ref:`Chronic Kidney Disease (CKD) / Impaired Kidney Function (IKF) <2017_cause_ckd>`
+
+* :ref:`Diabetes Mellitus (DM) / Fasting Plasma Glucose (FPG) <_2017_cause_diabetes_mellitus>`
+
+Risk-Outcome Relationships
+++++++++++++++++++++++++++
+
+Coverage Gap Framework
+++++++++++++++++++++++
+
+Eligible to Treatment Criteria
```

Comment:
Will we assign treatment to individuals who had treatment before? Is this a one-time treatment or continuing treatment over 5 years?

Reply:
Yes! We will assign treatment to individuals who had treatment before. In each scenario, this will look different. Also, it is a continuing treatment over 5 years. I'll add this information to the criteria before pushing to master.